src/sentry/testutils/pytest/relay.py

```diff
@@ -78,9 +76,13 @@ def relay_server_setup(live_server, tmpdir_factory):
     )
     assert redis_db == projectconfig_backend.cluster.connection_pool.connection_kwargs["db"]
 
+    with socket.socket() as sock:
+        sock.bind(("127.0.0.1", 0))
+        RELAY_PORT = sock.getsockname()[1]
```

Comment:
use `ephemeral-port-reserve` -- buck and I wrote it and it does this consistently

Reply:
(and without the race)

Reply:
:O